utils/test_helper.py

```diff
@@ -0,0 +1,183 @@
+from django.test import Client
+
+import os
+import random
+import string
+import filecmp
+import shutil
+
+from problem.models import Problem, Tag, Testcase, Submission
+from users.models import User
+from utils import config_info
+
+TESTCASE_PATH = config_info.get_config('path', 'testcase_path')
+TEST_PATH = ""
+
```

Comment:
The comment is just to separate testing code and functional code for maintenance but I think original design in Django is also ok now.